src/team.cpp

```diff
@@ -16,6 +17,13 @@ Team::~Team()
 
 }
 
+void Team::displayAll() const
+{
+	std::cout << aId << " - "
```

Comment:
qDebug ?

Reply:
std::cout << "Changed to qDebug()" << std::endl;